.ci/scripts/go-plus/tgc-tester-integration/test_tgc_integration.sh

```diff
@@ -12,7 +12,7 @@ github_username=modular-magician
 
 
 new_branch="auto-pr-$pr_number"
-git_remote=https://$github_username:$GITHUB_TOKEN@github.com/$github_username/$gh_repo
+git_remote=https://$github_username:$GITHUB_TOKEN_CLASSIC@github.com/$github_username/$gh_repo
```

Comment:
```suggestion
git_remote=https://github.com/$github_username/$gh_repo
```

Shouldn't need a token here since we're not pushing code.